packages/remirror__extension-events/src/events-extension.ts

```diff
@@ -204,6 +210,10 @@ export type HoverEventHandler = (
   event: MouseEvent,
   state: HoverEventHandlerState,
 ) => boolean | undefined | void;
+export type EditableEventHanlder = (
+  previousEditable: boolean,
+  editable: boolean,
+) => boolean | undefined | void;
```

Comment:
I guess we only need to accept one boolean value, since the other one is always the opposite. I prefer the current editable value.

Reply:
Seems like this function doesn't need to return a boolean value. It won't get used anyway.